linera-service/src/linera/main.rs

```diff
@@ -1305,14 +1307,14 @@ async fn run(options: ClientOptions) -> anyhow::Result<()> {
                 let chain = UserChain::make_initial(&mut rng, description, timestamp);
                 // Public "genesis" state.
                 let key = chain.key_pair.as_ref().unwrap().public();
-                genesis_config.chains.push((key, *initial_funding));
+                genesis_config_ref.chains.push((key, *initial_funding));
                 // Private keys.
                 chains.push(chain);
             }
-            genesis_config.write(genesis_config_path)?;
-            let mut wallet_state = options.create_wallet(genesis_config, *testing_prng_seed)?;
-            wallet_state.extend(chains);
-            wallet_state.save()?;
+            drop(genesis_config_ref);
```

Comment:
With `Persist::mutate`, we can't forget to persist the value.  The `drop` here is necessary because we move out of `genesis_config` — otherwise we'll have a compile error.  We could also scope `genesis_config_ref` to a block!